### PR TITLE
Better CN translations in various configuration

### PR DESCRIPTION
**Pull request by ghost**
And change bracket into single-byte one, to match others.

@Bluefissure @ShadyWhite Do you guys have time to review my translation? I am not sure if they are described correctly.

### CODE REVIEW

File: ui/eureka/eureka_config.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ UserConfig.registerOptions('eureka', {
       name: {
         en: 'Volume of the bunny pop sound (0-1)',
         ja: 'しあわせうさぎ出現音量 (0-1)',
+        cn: '幸运兔提示音量',
```
Comment (maiko-tan-coding) [inline ui/eureka/eureka_config.js:42]: Is "幸运兔" correct? I mean is it definitely the text in-game?

File: ui/jobs/jobs_config.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ UserConfig.registerOptions('jobs', {
         de: 'Verstecke das Käse Icon wenn Bufffood > Zeit (in Sekunden)',
         fr: 'Masquer l\'icône du fromage lorsque vous êtes repu > durée (en secondes)',
         ja: '飯効果の時間が不足したらチーズアイコンを表示する (秒)',
-        cn: '食物提醒图标隐藏时间（秒）',
+        cn: '当食物Buff时间大于该值时隐藏奶酪图标 (秒)',
```
Comment (maiko-tan-coding) [inline ui/jobs/jobs_config.js:51]: Hard to describe this option.

File: ui/jobs/jobs_config.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ UserConfig.registerOptions('jobs', {
         de: 'Minimum an Sekunden für einen Cooldown vor der Anzeige',
         fr: 'Nombre minimal de secondes avant l\'affichage du temps de recharge',
         ja: 'シナジースキルが使用可能前にアイコンを表示する (秒)',
-        cn: '显示团辅冷却倒计时最小时间',
+        cn: '当团辅技能的冷却时间小于该值时显示对应的团辅技能图标 (秒)',
```
Comment (maiko-tan-coding) [inline ui/jobs/jobs_config.js:103]: Hard to describe

File: ui/jobs/jobs_config.js [hunk 4]
```diff
@@ -126,7 +126,7 @@ UserConfig.registerOptions('jobs', {
         de: 'Höhe des Buff Icons (px)',
         fr: 'Hauteur des icônes de buff (pixel)',
         ja: 'シナジースキルのアイコンの高さ (pixel)',
-        cn: '团副监控图标高度（像素）',
+        cn: '团辅监控图标高度 (像素)',
```
Comment (maiko-tan-coding) [inline ui/jobs/jobs_config.js:129]: typo (*￣3￣)╭

File: ui/raidboss/raidboss_config.js [hunk 5]
```diff
@@ -1219,7 +1219,7 @@ UserConfig.registerOptions('raidboss', {
         fr: 'Utiliser le navigateur pour la synthèse vocale', // Machine Translation
         ko: '웹브라우저를 이용해서 TTS 작동시키기',
         ja: 'ブラウザでTTS',
-        cn: '使用浏览器进行文字转语音', // Machine translation
+        cn: '忽略ACT的设置，使用Cactbot自带的Google TTS功能（需联网）',
```
Comment (maiko-tan-coding) [inline ui/raidboss/raidboss_config.js:1222]: So it may not be usable in Mainland China

File: ui/eureka/eureka_config.js (hunk 1 above)
Comment (Bluefissure) [reply ui/eureka/eureka_config.js:42]: "幸福兔" is correct

Comment (maiko-tan-coding) [reply ui/eureka/eureka_config.js:42]: Okay～

**Comment by quisquous** [comment]
Let me know when you'd like me to take a look at this and merge it.  Thanks for translations!

File: ui/jobs/jobs_config.js (hunk 2 above)
Comment (Bluefissure) [reply ui/jobs/jobs_config.js:51]: Would `食物 Buff 剩余时间警报 (秒)` be better?

File: ui/jobs/jobs_config.js (hunk 3 above)
Comment (Bluefissure) [reply ui/jobs/jobs_config.js:103]: Would `团辅冷却完成预告 (秒)` be better?

File: ui/raidboss/raidboss_config.js (hunk 5 above)
Comment (Bluefissure) [reply ui/raidboss/raidboss_config.js:1222]: Well Google is banned in China ><

**Comment by MaikoTan** [comment]
> Two of the names can be more concise, rather than describing their functions.

Sure, and sorry for deleting my secondary account, this one is my primary account.

I would fix them soon.

File: ui/jobs/jobs_config.js (hunk 3 above)
Comment (MaikoTan) [reply ui/jobs/jobs_config.js:103]: But that would not describe what is the value used for.
Or `团辅技能冷却时间较长时隐藏图标（秒）` ?

File: ui/jobs/jobs_config.js (hunk 2 above)
Comment (MaikoTan) [reply ui/jobs/jobs_config.js:51]: Or `食物 Buff 剩余时间不足警报（秒）`？
I think the term "be expired soon" should be described in the options text.

File: ui/jobs/jobs_config.js (hunk 3 above)
Comment (Bluefissure) [reply ui/jobs/jobs_config.js:103]: I think you're making this too complicated, `预告` can show that cooldown is about to be done.

Comment (MaikoTan) [reply ui/jobs/jobs_config.js:103]: Yeah, maybe I am thinking too much. Thanks for pointing out.

**Comment by MaikoTan** [comment]
@quisquous I think it can be merge now. Applying for your review. (^o^)丿

**Comment by quisquous** [comment]
Thanks!!

**Pull request merged**